components/validation/validation-custom-mixin.js

```diff
@@ -1,3 +1,4 @@
+import { isCustomFormElement } from '../form/form-helper.js';
 
 export const ValidationCustomMixin = superclass => class extends superclass {
```

Comment:
Trying to keep all the things straight...  I feel like we have some "validation" word soup.  :)  How would you best describe this?  Is this a "custom validator mixin"?  Could this just as easily be called `ValidatorMixin`... after all, anything that extends it is effectively "custom" ya?  I don't mean to bike shed... initial glance I was (and maybe still am) unsure of what all the things are.